src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -646,16 +657,19 @@ function copyOptions() {
         return;
     }
     
-    // Feedback giver setup
+    // Previous feedback path setup
     var $prevGiver = $('select[name="givertype"]').eq(-2);
-    var $currGiver = $('select[name="givertype"]').last();
+    var $prevRecipient = $('select[name="recipienttype"]').eq(-2);
     
-    $currGiver.val($prevGiver.val());
+    // If previous feedback path was custom, there's no need to copy
+    if ($prevGiver.val() === 'CUSTOM' || $prevRecipient.val() === 'CUSTOM') {
+        return;
```

Comment:
Maybe for consistency's sake, we should copy the custom feedback path too; otherwise, I would recommend adding a prominent note to the header comments regarding this special case